stories/RefinementList.stories.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import React, { Component } from 'react';
 import orderBy from 'lodash.orderby';
```

Comment:
since it's a story and we have a dependency on lodash already, you can change this to `import orderBy from 'lodash/orderby';`. That way this devdep can go :)

Reply:
Actually no I explicitly use the `lodash.orderby` because Rollup messed up when `lodash` is present as a top level dependency on the Workspaces.